src/interceptors/multipart_request.js

```diff
@@ -1,4 +1,17 @@
-import _ from 'lodash';
+const _ = require('lodash');
+const FormData = require('form-data');
+
+/**
+   Wraps FormData's async getLength fn in a Promise
+*/
+const getLengthPromise = formData =>  new Promise((resolve, reject) => {
+  formData.getLength((err, length) => {
+    if (err) {
+      reject(err);
+    }
+    resolve(length);
+  });
+});;
```

Comment:
Any opinions, use this or `util.promisify`?

Reply:
I think this is  fine and clear. Naming suggestion: `getLengthPromise` => `getLength`, no need to have the return type in the name.